uaclient/tests/test_cli_enable.py

```diff
@@ -458,13 +461,13 @@ def test_beta_entitlement_instantiated_and_enabled_with_config_override(
             ent_name: m_entitlement_cls
         }
 
-        kwargs = {"allow_beta": False}
+        kwargs = {"allow_beta": False, "allow_disable": False}
         if silent_if_inapplicable is not None:
             kwargs["silent_if_inapplicable"] = silent_if_inapplicable
         ret = _perform_enable(ent_name, m_cfg, **kwargs)
 
         assert [
-            mock.call(m_cfg, assume_yes=False)
+            mock.call(m_cfg, assume_yes=False, allow_disable=False)
```

Comment:
I'm -1 on "allow_disable" as it's very unclear what the disable/allow are speaking to here.